core-services/prow/02_config/openshift/machine-api-provider-ibmcloud/_prowconfig.yaml

```diff
@@ -0,0 +1,74 @@
+tide:
+  queries:
+  - includedBranches:
+    - openshift-4.10
+    - openshift-4.9
+    - release-4.10
+    - release-4.9
```

Comment:
i'm not sure we should include these branches, we would need to bring them over.